Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -163,11 +165,20 @@
 #import <WebCore/PromisedAttachmentInfo.h>
 #endif
 
+#if USE(APPLE_INTERNAL_SDK)
+#import <AXRuntime/AXDefines.h>
+#import <AXRuntime/AXNotificationConstants.h>
+#endif
```

Comment:
Our modern practice is to put these into SPI.h headers, not just into the one file where they are used. Sprinkling them into different files leads to a mess over time, and we would like to avoid that.